api/tx_args.go

```diff
@@ -445,3 +447,43 @@ func (args *AccountUpdateTxArgs) toTransaction() (*types.Transaction, error) {
 
 	return tx, nil
 }
+
+// isReverted checks given error is vm.ErrExecutionReverted
```

Comment:
Why did you decide to move those logic to here?
How about in `api_public_blockchain.go` file?